dev-lang/gnucobol/gnucobol-3.0_rc1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PV=$(ver_cut 1-2)
+MY_P="${PN}-${MY_PV}-rc1"
+
+DESCRIPTION="an open-source COBOL compiler"
+HOMEPAGE="https://open-cobol.sourceforge.io"
+SRC_URI="mirror://sourceforge/open-cobol/${MY_P}.tar.gz"
+
+LICENSE="GPL-3 LGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="berkdb nls readline"
+
+RDEPEND="dev-libs/gmp:0
```

Comment:
Prefer sorting unconditional deps first, then all conditionals. It makes it easier to verify them later.